shell/browser/browser_mac.mm

```diff
@@ -373,15 +373,15 @@ void RemoveFromLoginItems() {
       orderFrontStandardAboutPanelWithOptions:options];
 }
 
-void Browser::SetAboutPanelOptions(const base::DictionaryValue& options) {
+void Browser::SetAboutPanelOptions(base::DictionaryValue options) {
   about_panel_options_.Clear();
 
-  for (const auto& pair : options) {
-    std::string key = pair.first;
-    const auto& val = pair.second;
+  for (auto& pair : options) {
+    std::string& key = pair.first;
+    std::unique_ptr<base::Value>& val = pair.second;
```

Comment:
Any reason for this to be a reference since we are eventually moving it ?

Reply:
Didn't realize this is unneeded. 👍